manifests/init.pp

```diff
@@ -35,6 +35,10 @@
   String $map_source,
   Optional[String] $service = undef,
   Optional[String] $service_provider = undef,
+  String $username = 'tftp',
+  String $address = '',
+  Integer $port = 69,
+  String $options = '--secure',
```

Comment:
Perhaps this should default to an empty string and only be set for Debian in `data/Debian.yaml`?

Reply:
done